sceptre/config/reader.py

```diff
@@ -163,14 +180,22 @@ def read(self, rel_path, base_config=None):
         if base_config:
             config.update(base_config)
 
-        # Check is file exists, but ignore config.yaml as can be inherited.
+        # Check if file exists, but ignore config.yaml as can be inherited.
         if not path.isfile(abs_path) and not filename.endswith("config.yaml"):
             raise ConfigFileNotFoundError(
                 "Config file \"{0}\" not found.".format(rel_path)
             )
 
         # Parse and read in the config files.
-        config.update(self._recursive_read(directory_path, filename))
+        this_config = self._recursive_read(directory_path, filename)
+
+        if "dependencies" in config or "dependencies" in this_config:
+            this_config['dependencies'] = strategies.list_join(
```

Comment:
Should this use the strategy for `dependencies` that is defined in `CONFIG_MERGE_STRATEGIES` rather than always doing `list_join`?

Reply:
yes good spot